misc/acrn-config/xmls/board-xmls/apl-up2.xml

```diff
@@ -37,8 +37,8 @@
 	00:15.0 USB controller: Intel Corporation Celeron N3350/Pentium N4200/Atom E3900 Series USB xHCI (rev 0b)
 	Region 0: Memory at 91500000 (64-bit, non-prefetchable) [size=64K]
 	00:15.1 USB controller: Intel Corporation Device 5aaa (rev 0b)
-	Region 0: Memory at 91000000 (64-bit, non-prefetchable) [size=2M]
-	Region 2: Memory at 91537000 (64-bit, non-prefetchable) [size=4K]
+	Region 0: Memory at 91000000 (64-bit, non-prefetchable) [disabled] [size=2M]
```

Comment:
How is the BAR information used by config tool?

Reply:
This is for vbar_base usage in struct acrn_vm_pci_dev_config.
If ve820 layout could match pe820 on pci config space, we can use pbar base directly as vbar's.